javascript/modal.js

```diff
@@ -0,0 +1,14 @@
+const modelAvtive = document.querySelector("#btn-modal-active")
+const modelBox = document.querySelector("#model-box")
+const modalClose = document.querySelector("#model-close")
+
+console.log(modelAvtive);
+
+modelAvtive.addEventListener("click", () => {
+    modelBox.style.display="block"
+})
+
+modalClose.addEventListener('click', () => {
+    modelBox.style.display="none"
+
+})
```

Comment:
here you use es5 and es6 that is i don't like you should use es6 all or es5 only one. that is not good.
please remove extra space

Reply:
ok thank you for suggestion ! i will do